polynote-spark/src/main/scala/polynote/kernel/LocalSparkKernel.scala

```diff
@@ -119,7 +121,15 @@ class LocalSparkKernelFactory extends Kernel.Factory.LocalService {
     settings
   }
 
+  private def loadCredentials(credentials: Credentials): URIO[Logging, List[DirectCredentials]] = credentials.coursier match {
+    case Some(Credentials.Coursier(path)) =>
+      Task(CoursierCredentials(new File(path), optional = false).get().toList)
```

Comment:
`Credentials#get()` from coursier does blocking IO, so it should be in `effectBlocking()` rather than `Task()` (so this would have to return `URIO[Blocking with Logging, List[DirectCredentials]` – I'd just make it `URIO[BaseEnv, List[DirectCredentials]` which includes both of those)